CONTRIBUTING.md

```diff
@@ -62,28 +62,34 @@ discuss pending contributions in order to find a resolution. It is expected that
 small minority of issues be brought to the WG for resolution and that discussion and
 compromise among collaborators be the default resolution mechanism.
 
-# Becoming a Collaborator
+# Becoming a Member and Collaborator
 
-All contributors who land a non-trivial contribution should be on-boarded in a timely manner,
+To become a member of the Community Committee, start participating! Attend the bi-weekly meetings, 
+file issues and pull requests, and provide insight via GitHub. Request to become an Observer by 
```

Comment:
> Not sure if we need to provide more guidance here, but I think it would be helpful. What should I, as a person looking to start contributing, file issues and PRs about/for?

I'm kinda weary about explicitly specifying this because a) it means we need someone to manage this process and we're typically limited time-wise (providing a well-defined trail means we need people to do trail upkeep) and b) what needs to be done will change over time anyways and listing everything out might actually box us in.

That said, we should probably be making use of the "good first contribution" label. I suspect people will want to get involved for one of two reasons: 1) they have a specific goal they want to accomplish, in which case they will know what issue to open or 2) they don't have a specific goal and want to take on whatever work we have available (so, "good first contribution").

I think it could make sense to add something like "investigate issues tagged as `good first contribution1" to the lsit.